controllers/vspherecluster_controller.go

```diff
@@ -56,48 +55,13 @@ type VSphereClusterReconciler struct {
 
 // Reconcile ensures the back-end state reflects the Kubernetes resource state intent.
 func (r *VSphereClusterReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr error) {
```

Comment:
Hi @yastij,

I'm not sure I want the context initialization of this controller refactored as part of *this* PR. I don't mind handling it later, but I'd like to reduce this PR to just what is required for the LB. What do you think?